### PR TITLE
RCAL-853: Give VA scale factor a reasonable default

### PR DESCRIPTION
**Pull request by stscieisenhamer**
WIP [RCAL-853](https://jira.stsci.edu/browse/RCAL-853)

Main PR: [PR#1354](https://github.com/spacetelescope/romancal/pull/1354/files)

This PR gives the `velocity_aberration.scale_facter` meta a default working value of 1.0

## Tasks
- [x] update or add relevant tests
- [x] update relevant docstrings and / or `docs/` page
- [ ] Does this PR change any API used downstream? (if not, label with `no-changelog-entry-needed`)
  - [ ] write news fragment(s) in `changes/`: `echo "changed something" > changes/<PR#>.<changetype>.rst` (see below for change types)
  - [ ] [start a `romancal` regression test](https://github.com/spacetelescope/RegressionTests/actions/workflows/romancal.yml) with this branch installed (`"git+https://github.com/<fork>/roman_datamodels@<branch>"`)

<details><summary>news fragment change types...</summary>

- ``changes/<PR#>.feature.rst``: new feature
- ``changes/<PR#>.bugfix.rst``: fixes an issue
- ``changes/<PR#>.doc.rst``: documentation change
- ``changes/<PR#>.removal.rst``: deprecation or removal of public API
- ``changes/<PR#>.misc.rst``: infrastructure or miscellaneous change
</details

### CODE REVIEW

**Review by schlafly — COMMENTED** [review]
This looks good to me but can you run regtests?  I think those all basically pass through the metadata from the L1s and so probably aren't sensitive to this, but if they are we should know about it.

**Comment by stscieisenhamer** [comment]
The doc issue must be something upstream. I can investigate or leave to others.

File: src/roman_datamodels/maker_utils/_common_meta.py [hunk 1]
```diff
@@ -254,7 +254,7 @@ def mk_velocity_aberration(**kwargs):
     vab = stnode.VelocityAberration()
     vab["ra_reference"] = kwargs.get("ra_reference", NONUM)
     vab["dec_reference"] = kwargs.get("dec_reference", NONUM)
-    vab["scale_factor"] = kwargs.get("scale_factor", NONUM)
+    vab["scale_factor"] = kwargs.get("scale_factor", 1)
```
Comment (braingram) [inline src/roman_datamodels/maker_utils/_common_meta.py:257]: Are these expected to be reasonable? I assumed they were intentionally nonsense values that the pipeline replaces.

**Review by PaulHuwe — APPROVED** [review]
LGTM

File: src/roman_datamodels/maker_utils/_common_meta.py (hunk 1 above)
Comment (schlafly) [reply src/roman_datamodels/maker_utils/_common_meta.py:257]: The reason to give a reasonable default here is that we are going to start actually using this value.  The scale factor makes a pretty small effect (fraction of a pixel or so), so you don't really want things to explode if people don't set it.  Something like ra/dec doesn't have a reasonable default that's right to within a fraction of a pixel all of the time, so it's reasonable to put a crazy number there, but lots of people won't want to think about the scale factor and my preference is not to force them to.

**Review by schlafly — APPROVED** [review]

File: src/roman_datamodels/maker_utils/_common_meta.py (hunk 1 above)
Comment (braingram) [reply src/roman_datamodels/maker_utils/_common_meta.py:257]: Thanks. It's helpful to know that these are sometimes valid defaults.

**Pull request merged**